app/src/pages/inside/stepPage/modals/makeDecisionModal/selectDefectManually/selectDefectManually.jsx

```diff
@@ -41,13 +41,23 @@ import styles from './selectDefectManually.scss';
 
 const cx = classNames.bind(styles);
 
+const BREAKPOINTS = {
+  HIDE_IGNORE_AA: 810,
+  SCREEN_XS_MAX: 767,
```

Comment:
We already have this value here `app / src / common / constants / screenSizeVariables.js.`
I think you should put all variables that connect with breakpoints in the separate file

Reply:
I have removed all `BREAKPOINT`, add standard breakpoints to constants and using it in all necessary files